### PR TITLE
3.3.0: adds release note for backup performance

### PR DESCRIPTION
**Pull request by ryanwittrup**
Includes new image with charts showing faster backup time and smaller backup artifact

[#187597108](https://www.pivotaltracker.com/story/show/187597108)

Authored-by: Ryan Wittrup <rwittrup@vmware.com>

Which other branches should this be merged with (if any)?
NA

### CODE REVIEW

File: release-notes.html.md.erb [hunk 1]
```diff
@@ -30,6 +30,16 @@ This release includes the following changes:
 * The bootstrap errand self-selects a high-availability cluster instance node on which to run, and no longer requires the `--instance` flag.
 * A number of high and critical CVEs are addressed by updating components to the latest versions.
 
+### Performance Improvements
+With the change to the compression algorithm, there has been a substantial improvement to the backup performance.
```
Comment (malba) [inline release-notes.html.md.erb:34]: Suggestion - can we change to?:
"With the introduction of a more efficient compression algorithm, backup performance has substantially improved."

File: release-notes.html.md.erb [hunk 2]
```diff
@@ -30,6 +30,16 @@ This release includes the following changes:
...
+
+Benchmark tests were performed against an arbitrary data set of around 150 GiB, and using a range of compute options (2 core / 2GB, 4 core / 16GB, 8 core / 64GB). Given those conditions, upgrading to version 3.3.0 of <%= vars.product_full %> resulted in approximately 8X faster backups and a 13% smaller backup artifact.
```
Comment (malba) [inline release-notes.html.md.erb:36]: Suggestion - can we change to?:
"Benchmark tests were performed using a publicly available dataset sample of around 15 GiB, using a range of compute configurations (2 core / 2GB, 4 core / 16GB, 8 core / 64GB). Under those parameters, upgrading to version 3.3.0 of <%= vars.product_full %> resulted in approximately 8X faster backups, and 13% smaller backup artifacts when compared against version 3.2.1."

**Review by malba — COMMENTED** [review]
Suggesting some minor edits to the narrative, other than that looks good!

**Comment by anita-flegg** [comment]
FYI @HenryBorys

File: release-notes.html.md.erb (hunk 1 above)
Comment (ryanwittrup) [reply release-notes.html.md.erb]: Should be addressed with most recent commit - but doublecheck and let me know

**Pull request merged**